### PR TITLE
Javascript assignment 

### PR DESCRIPTION
**Pull request by emtec-damodhar-jadhao**
Javascript Assignment completed

### CODE REVIEW

File: 1_September/js_assignment.html [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+  </head>
+  <body>
+    <h2>Javascript Assignment</h2>
+
+
+
+    <script>
+        //1) Create a function that takes two numbers as arguments and returns their sum.
+        // function addition(num1,num2){
+        //     console.log(num1+num2)
```
Comment (nitish-poojari) [inline 1_September/js_assignment.html]: You have been asked to return the value to caller and print in div tag

Comment (emtec-damodhar-jadhao) [reply 1_September/js_assignment.html]: Updated File As Per Instruction

File: 1_September/js_assignment.html [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+
+
+
+    <script>
+        //1) Create a function that takes two numbers as arguments and returns their sum.
+        // function addition(num1,num2){
+        //   return num1+num2    
+        // }
+        // const test1=console.log(addition(3, 2))
+        // const test2=console.log(addition(-3, -6))
```
Comment (nitish-poojari) [inline 1_September/js_assignment.html:20]: create seperate script folder and create script file for each function.

Comment (emtec-damodhar-jadhao) [reply 1_September/js_assignment.html:20]: created new PR

**Review by nitish-poojari — APPROVED** [review]

**Pull request merged**